tests/rados/test_osd_inconsistency_pg.py

```diff
@@ -70,6 +80,22 @@ def run(ceph_cluster, **kw):
             else:
                 log.error("Inconsistent object is not exists in the objects list")
                 return 1
+        osd_map_output = rados_obj.get_osd_map(pool=pool_name, obj=oname)
+        primary_osd = osd_map_output["acting_primary"]
+        # Executing the fsck on the OSD
+        bluestore_obj.run_consistency_check(primary_osd)
+        # Reparing the OSD using the bluestore tool
+        bluestore_obj.repair(primary_osd)
```

Comment:
Let's add debug level here to print all the outputs from the CBT tool

Reply:
The outputs are added.